ansibullbot/utils/component_tools.py

```diff
@@ -136,8 +138,8 @@ def index_files(self):
             if os.path.isdir(fn):
                 continue
             mname = os.path.basename(fn)
-            mname = mname.replace('.py', '').replace('.ps1', '')
-            if mname.startswith('__'):
+            mname = mname.replace(b'.py', b'').replace(b'.ps1', b'')
+            if mname.startswith(b'__'):
```

Comment:
You should establish some convention to show that self.gitrepo.module_files is a list of byte strings.  In ansible proper we name variables that hold byte strings with a b_ prefix.  So: self.gitrepo.b_module_files

(If you don't do this, at some point someone will put text strings in instead of byte strings and then this code will break.)

Reply:
That's a very good point. I didn't want to remake everything in this PR though.